.github/pull_request_template.md

```diff
@@ -0,0 +1,5 @@
+Please include:
+
+* background to your PR, and link any issues solved if necessary
+* any testing that you've done to ensure your PR is working
+* add your changes to the CHANGELOG.md at the top, under the "Unreleased" section
```

Comment:
It's a template, right? Not instructions beforehand? So maybe a checklist instead, and some sections to fill in? As you said, we can discuss this later if we want. Just brainstorming out loud.

```suggestion
In this PR, I have:

* [ ] Added tests for any new features or bug fixes (if relevant)
* [ ] Added my changes to the CHANGELOG.md at the top, under the "Unreleased" section
* [ ] Ensured issues that this PR closes will be [automatically closed](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue)

### Reason for this PR

<Insert reasons for this PR>

### Links to any solved or related issues

<Insert links>

### Any manual testing I have done to ensure my PR is working

<Insert description>
```